From 9f44de7f84cd8f8e0b10105d80f63e95ba73f015 Mon Sep 17 00:00:00 2001
From: Thierry Reding <thierry.reding@avionic-design.de>
Date: Wed, 12 Sep 2012 09:29:33 +0200
Subject: [PATCH 2/3] Do not distribute exsltconfig.h

This fixes a problem when building out of tree. If exsltconfig.h is
shipped in the tarball, the shipped version will be used instead of
the version generated in the build tree because the source tree
appears earlier in the include path.

Signed-off-by: Thierry Reding <thierry.reding@avionic-design.de>
---
 libexslt/Makefile.am | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/libexslt/Makefile.am b/libexslt/Makefile.am
index d0a7f92..d2566be 100644
--- a/libexslt/Makefile.am
+++ b/libexslt/Makefile.am
@@ -10,9 +10,11 @@ exsltincdir = $(includedir)/libexslt
 
 exsltinc_HEADERS =                      \
 	exslt.h				\
-	exsltconfig.h			\
 	exsltexports.h
 
+nodist_exsltinc_HEADERS =		\
+	exsltconfig.h
+
 libexslt_la_SOURCES =                   \
 	exslt.c				\
 	common.c			\
diff --git a/libexslt/exsltconfig.h b/libexslt/exsltconfig.h
deleted file mode 100644
index cfacbe3..0000000
--- a/libexslt/exsltconfig.h
+++ /dev/null
@@ -1,73 +0,0 @@
-/*
- * exsltconfig.h: compile-time version informations for the EXSLT library
- *
- * See Copyright for the status of this software.
- *
- * daniel@veillard.com
- */
-
-#ifndef __XML_EXSLTCONFIG_H__
-#define __XML_EXSLTCONFIG_H__
-
-#ifdef __cplusplus
-extern "C" {
-#endif
-
-/**
- * LIBEXSLT_DOTTED_VERSION:
- *
- * the version string like "1.2.3"
- */
-#define LIBEXSLT_DOTTED_VERSION "1.1.27"
-
-/**
- * LIBEXSLT_VERSION:
- *
- * the version number: 1.2.3 value is 10203
- */
-#define LIBEXSLT_VERSION 816
-
-/**
- * LIBEXSLT_VERSION_STRING:
- *
- * the version number string, 1.2.3 value is "10203"
- */
-#define LIBEXSLT_VERSION_STRING "816"
-
-/**
- * LIBEXSLT_VERSION_EXTRA:
- *
- * extra version information, used to show a CVS compilation
- */
-#define	LIBEXSLT_VERSION_EXTRA ""
-
-/**
- * WITH_CRYPTO:
- *
- * Whether crypto support is configured into exslt
- */
-#if 1
-#define EXSLT_CRYPTO_ENABLED
-#endif
-
-/**
- * ATTRIBUTE_UNUSED:
- *
- * This macro is used to flag unused function parameters to GCC
- */
-#ifdef __GNUC__
-#ifdef HAVE_ANSIDECL_H
-#include <ansidecl.h>
-#endif
-#ifndef ATTRIBUTE_UNUSED
-#define ATTRIBUTE_UNUSED __attribute__((unused))
-#endif
-#else
-#define ATTRIBUTE_UNUSED
-#endif
-
-#ifdef __cplusplus
-}
-#endif
-
-#endif /* __XML_EXSLTCONFIG_H__ */
-- 
1.7.12

